reana_server/cli.py

```diff
@@ -51,6 +51,19 @@
 .. code-block :: bash
 
     $ flask users get --admin-access-token=<token>
+
+To export, all users in current REANA cluster:
+
+.. code-block :: bash
+
+    # users will be exported in CSV format
+    $ flask export --admin-access-token=<token>
+
+To import users to current REANA cluster:
+
+.. code-block :: bash
+
+    $ flask import --file <users.txt> --admin-access-token=<token>
```

Comment:
And then here we can use the same name:

```suggestion
    $ flask import --file users.csv --admin-access-token=<token>
```